recipes/gdist/meta.yaml

```diff
@@ -0,0 +1,47 @@
+package:
+  name: gdist
+  version: "1.0.3"
+
+source:
+  md5: 98547eac8e7c0b5a7a600e12ae5c9c4c
+  fn: 'gdist-1.0.3.tar.gz'
+  url: https://pypi.io/packages/source/g/gdist/gdist-1.0.3.tar.gz
```

Comment:
You should use jinja variables for the version here, to make it easier to update in the future: `{% set version = '1.0.3' %}` at the top, and change to `gdist-{{ version }}.tar.gz` here, like in the other recipes. `fn` should change to `gdist-{{ version }}.tar.gz` too.